lisa/transformers/kernel_source_installer.py

```diff
@@ -27,7 +26,12 @@ class BaseModifierSchema(schema.TypedSchema, schema.ExtendableSchemaMixin):
 @dataclass_json()
 @dataclass
 class BaseLocationSchema(schema.TypedSchema, schema.ExtendableSchemaMixin):
-    ...
+    kernel_config_file: str = field(
```

Comment:
This field needs to be documented properly. Please add a comment that addresses the following questions:
1. Is this a path on the local node (where lisa is running) or remote node?
2. Is it an absolute path or a path relative to the root of the kernel source tree?

Reply:
Comment is added, Path is absolute now

Reply:
@anirudhrb  As we discussed, kernel config file will be relative path to the kernel source code path

Reply:
updated the comment